package/MDAnalysis/topology/tpr/utils.py

```diff
@@ -74,6 +74,8 @@
     Impropers,
 )
 
+global_precision = 4
```

Comment:
suggestions for tracking TPR file precision in a way that doesn't use a `global` are appreciated